packages/kotti-ui/source/kotti-field-password/KtFieldPassword.vue

```diff
@@ -47,7 +47,7 @@ export default defineComponent({
 			field,
 			inputProps: computed(
 				(): Partial<HTMLInputElement> & {
-					class: object
```

Comment:
this was inferred, but it could be `string[] | Record<string, boolean>` - would the rule object if you declare the type the way vue allows class to be? question

Reply:
answer, no inference, it just detected the `object`